[FIX] Remove last_request_end_time predicate

amachanic opened the pull request:
I think this will fix various bugs. On the flip side, I have no idea what problems it will cause. This predicate was in place for about a decade and I can't remember why I put it in place.

erikdarlingdata commented:
@amachanic not sure what happened here, but essentially the whole PR is a conflict. It might be easier to re-base and make the one change again.

amachanic commented:
> @amachanic not sure what happened here, but essentially the whole PR is a conflict. It might be easier to re-base and make the one change again.

I removed all of the tabs and switched to spaces.

erikdarlingdata commented:
> > @amachanic not sure what happened here, but essentially the whole PR is a conflict. It might be easier to re-base and make the one change again.
> 
> I removed all of the tabs and switched to spaces.

Yeah, I caught that PR, but the diff wasn't loading here for a bit. Would you prefer to rebase with the spaces commit in place or just resolve this one?
